src/SCRIPTS/BF/ui_init.lua

```diff
@@ -18,6 +11,22 @@ local function init()
         data_init = nil
         apiVersionReceived = true
         collectgarbage()
+    elseif not mcuId then
+        if apiVersion >= 1.042 then
+            lcd.drawText(6, radio.yMinLimit, "Waiting for unique device ID")
+            getMCUId = getMCUId or assert(loadScript("mcu_id.lua"))()
+            if getMCUId() then
+                getMCUId = nil
+                local vtxTables = loadScript("/BF/VTX/"..mcuId..".lua")
+                if vtxTables and vtxTables() then
+                    vtxTablesReceived = true
+                    vtxTables = nil
+                end
+                collectgarbage()
+            end
+        else
+            mcuId = model.getInfo().name
+        end
```

Comment:
Not sure if storing the model name in ```mcuId``` is the best thing to do. I would combine the conditionals like 
```elseif apiVersion >= 1.042 and not mcuId then```
and leave ```mcuId = nil```for older versions.
This would require loading vtx tables or defaults in vtx.lua based on ```apiVersion``` like this:
```
local vtx_tables
if apiVersion >= 1.042 then
    vtx_tables = assert(loadScript("/BF/VTX/"..mcuId..".lua"))()
else
    vtx_tables = assert(loadScript("/BF/VTX/vtx_defaults.lua"))()
end
```

Can we change the message to just "Waiting for device ID"? That would make it fit nicely on the 128x64 screens too.